### ISSUE TITLE
Trouble installing on Gentoo 4.4.6, glibc-2.23-r3

### ISSUE THREAD

**Issue by kub1x** [lovell/sharp#756]
During `npm install` I get following: 
```
ERROR: ldd (Gentoo 2.23-r3 p7) 2.23 requires manual installation - please see http://sharp.dimens.io/en/stable/install/
gyp: Call to 'LDD_VERSION="ldd (Gentoo 2.23-r3 p7) 2.23
Copyright (C) 2016 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper." node -e "require('./binding').download_vips()"' returned exit status 1 while in binding.gyp. while trying to load binding.gyp
```
And when running the obvious error: 
```
Error: Cannot find module '../build/Release/sharp.node'
    at (...)
```
I **have** `vips` installed globally, but it's binary is `vips` not `vips-cpp`.
```
$ pkg-config --modversion vips
7.38.5
```
Can I somehow configure `sharp` to look for that binary instead?

**Comment by lovell** [comment]
Hello, it looks like the libc version detection logic is failing on Gentoo.

The following change to [binding.js](https://github.com/lovell/sharp/blob/master/binding.js#L76) should "fix" this. Are you able to test locally?
```diff
- if (/(glibc|gnu libc)/i.test(lddVersion)) {
+ if (/(glibc|gnu libc|gentoo)/i.test(lddVersion)) {
```

I'll attempt to add Gentoo to the Docker-based [packaging tests](https://github.com/lovell/sharp/blob/ridge/packaging/test-linux-x64.sh) as I'm sure others will run into this. Thanks for reporting!

**Comment by lovell** [comment]
I had a quick attempt at adding Gentoo to the packaging tests but sadly it's not got any easier to use since I last tried it in ~2002.

Any hints for how to install nodejs on the [gentoo/stage3-amd64](https://hub.docker.com/r/gentoo/stage3-amd64/) Docker image would be most welcome :)

**Comment by kub1x** [comment]
Ok. I tried your change and it does help with the installation. 
The tests passed and manually calling the API does the job, so I created a PR for it here #760

I still have some troubles resizing images in my web server, but that would prob'ly be some stupid mistake. I'll give it a bit more digging. 
Thanks alot!

**Comment by lovell** [comment]
Thanks for the PR, this will be included in v0.18.0.

> From what I imagine: get that docker => instal nodejs => instal sharp => run tests.. right?

Yes please. It looks like it's not quite as simple as `emerge nodejs` so I'll try your `nvm` suggestion when I get a moment.

**Comment by lovell** [comment]
v0.18.0 now available, thanks for reporting this.
